src/pages/build-apps/examples/heystack.md

```diff
@@ -0,0 +1,482 @@
+of likes). Note that the message itself isn't stored in a contract variable, the frontend application reads the content
+of the message directly from the transaction on the blockchain.
+
+```clarity
+;;
+;; Public functions
+(define-public (send-message (content (string-utf8 140)))
+  (let ((id (unwrap! (increment-content-index) (err u0))))
+    (print { content: content, publisher: tx-sender, index: id })
+    (map-set like-state
+      { content-index: id }
+      { likes: u0 }
+    )
+    (map-set publisher-state
+      { content-index: id }
+      { publisher: tx-sender }
+    )
+    (transfer-hey u1 HEY_TREASURY)
+  )
+)
+```
+
+The `send-message` method accepts a utf-8 string with a maximum length of 140 characters. The method defines an internal
+variable `id` using the `let` function and assigns the next content ID to that variable by calling the
+`increment-contract-index` method of the contract. The value assignment of this variable is bounded by the [`unwrap!`][]
+function, which returns an error and exits the control-flow if the `increment-contract-index` function isn't
+successfully called.
+
+The method then assigns `u0` likes to the content in the `like-state` data map, and adds the principal address to the
+`publisher-state` data map using the [`map-set`][] function. Finally, the private method `transfer-hey` is called to
+transfer 1 $HEY token from the message sender to the $HEY treasury address stored in the `HEY_TREASURY` constant.
+
+```clarity
+(define-public (like-message (id uint))
+  (begin
+    ;; cannot like content that doesn't exist
+    (asserts! (>= (var-get content-index) id) (err ERR_CANNOT_LIKE_NON_EXISTENT_CONTENT))
+    ;; transfer 1 HEY to the principal that created the content
+    (map-set like-state
+      { content-index: id }
+      { likes: (+ u1 (get likes (unwrap! (get-like-count id) (err u0)))) }
+    )
+    (transfer-hey u1 (unwrap-panic (get-message-publisher id)))
+  )
+)
+```
+
+The `like-message` method accepts a content ID. The method checks that the ID is lower than the current content ID using
+the [`asserts!`][] function, to verify that the provided ID is a valid ID. If the [`asserts!`][] assessment is `false`,
+the method returns an error code. If the ID is valid, the method performs a [`map-set`][] to look up the content in the
+`like-state` data map and add a like to the value stored in the map. Once again, the [`unwrap`][] function is used to
+ensure that an invalid value isn't stored in the map.
+
+The `hey.clar` contract provides some additional functions for working with the $HEY token contract, discussed in the
+next section.
+
+### Token contract
+
+Heystack creates a native fungible token for use in the application. When a user authenticates with Heystack, they're
+automatically eligible to claim 100 $HEY tokens to allow them to start messaging.
+
+[SIP-010][] defines the fungible token started on Stacks, which allows Stacks compatible wallets to handle fungible
```

Comment:
```suggestion
[SIP-010][] defines a fungible token started on Stacks, which allows Stacks compatible wallets to handle fungible
```

Reply:
This is actually meant to read `SIP-010 defines the fungible token standard on Stacks`. Will fix the typo.